light_curves/multiband_lc.ipynb

```diff
@@ -546,7 +537,7 @@
    "outputs": [],
    "source": [
     "# Save the data for future use with ML notebook\n",
-    "#parquet_savename = 'data/df_lc_051123.parquet.gzip'\n",
+    "#parquet_savename = 'data/df_lc_090723_yang.parquet.gzip'\n",
     "#parallel_df_lc.data.to_parquet(parquet_savename)\n",
```

Comment:
Why does this file name end with ".gzip", does anyone know? I see two issues with this:

1. This file is not gzip-ed -- it's just a regular Parquet file. Thus it's recommended that the name end with ".parquet" because some generic readers can use this to automatically determine the file format.
2. The content in Parquet files is generally compressed on a per-column basis (which is different than compressing the entire file together). gzip is an option for this type of compression, but it's not what's being used here. The `to_parquet` call is using the default compression, which is snappy not gzip.

Reply:
done